DEVELOPER.md

```diff
@@ -242,8 +310,11 @@ We use the homegrown [`test-group`](https://crates.io/crates/test-group) crate t
   - `datafusion` - tests that use Apache DataFusion
   - `spark` - tests that use Apache Spark
   - `flink` - tests that use Apache Flink
+- `database` - for tests that involve any database interaction, subsequently grouped by:
+  - `mysql` - tests that use MySQL/MariaDB
+  - `postgres` - tests that use PostreSQL
```

Comment:
I think this directory lost its purpose. Lets move `database-common` to `utils` and `migrations` to root?

Reply:
Reorganized